### ISSUE TITLE
Code left in server even after removal of the plugin???

### ISSUE THREAD

**Issue by Yomamaeatstoes** [JT122406/HungerGames#22]
**Describe the bug**

Even after you remove the jar some things still leanger
**To Reproduce**

Delete Hunger Games

**Expected behavior**

It does not linger after removal of the plugin

**Screenshots/Errors**

https://timings.aikar.co/dev/?id=27e5e523e80c47fab6784fb8aeba8b55

https://spark.lucko.me/jmzfwrs3mY

Hunger Games was removed, but it is still there in the server. I was told " It appears that it may use Scoreboards or other built-in Minecraft features, which may be left in the world/worlds upon removal, unfortunately. "
**Version Info:** 1.19.2
 - **Plugin Version:**  5.2.3
 - **Server Version/Platform**  

**Additional context**

**Comment by Yomamaeatstoes** [comment]
This could still be partly true, but I found out that one of the players warped over to the Hunger Games map and set off a bunch of TNT when a creeper blew up destroying the Hunger Games spawn. So I think that the Hunger Games mentioned in the timings report is the Hunger Games spawn, and not the actual Hunger Games plugin. I wanted to mention this before hand once I found out because I thought It would be important to know. Thank you for your patience, and help you have given. Keep up the great work!
